app/routines/freeFlight/views/Fly.js

```diff
@@ -104,8 +104,8 @@ define(['app', 'backbone', 'JST', 'q', 'leaflet-dist', 'bootstrap-slider', 'unde
                 // Detect when system has landed, then instruct disarm.
                 this.model.platform.on('status:standby', function() {
```

Comment:
the trigger for `status:standby` is located [here](https://github.com/arctic-fire-development/dapper-gcs/blob/paths/app/Models/Platform.js#L120)